2-projects/1-todo-list/script.js

```diff
@@ -1,15 +1,67 @@
 function populateTodoList(todos) {
   let list = document.getElementById("todo-list");
   // Write your code to create todo list elements with completed and delete buttons here, all todos should display inside the "todo-list" element.
+  for (let i = 0; i < todos.length; i++) {
+    const listMember = document.createElement("li");
+    listMember.className =
+      "list-group-item d-flex justify-content-between align-items-center";
+    listMember.id = "list" + i;
+    const textListMember = document.createTextNode(`${todos[i].task} `);
+    listMember.appendChild(textListMember);
+
+    const mySpan = document.createElement("span");
```

Comment:
Can you think of a better name that `mySpan`? What is this element going to be used for?